HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Resources/PaymentReview.storyboard

```diff
@@ -46,7 +46,7 @@
                                                 </collectionViewFlowLayout>
                                                 <cells>
                                                     <collectionViewCell opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" reuseIdentifier="pageCellIdentifier" id="c1L-5D-aNE" customClass="PageCollectionViewCell" customModule="GiniHealthSDK">
-                                                        <rect key="frame" x="0.0" y="20" width="348" height="408.66666666666669"/>
+                                                        <rect key="frame" x="0.0" y="17" width="348" height="408.66666666666669"/>
```

Comment:
@ValentinaIancu-Gini could you please check if you need this change - seems like not.

Reply:
It was automatically modified. Should not affect anything.

Reply:
All good - just for double check :)